src/main/java/de/doubleslash/usb_led_matrix/view/AuthenticationView.java

```diff
@@ -143,6 +144,7 @@ void setTeamsID(ActionEvent event) {
          p.setProperty("tenant_id", tenantIDTextField.getText());
          p.setProperty("client_id", clientIDTextField.getText());
          p.store(fw, "Sample comments");
+         fw.close();
```

Comment:
looks weird that it is only closed when everything before succeeded. "try with" syntax would be better

Reply:
Done.